### PR TITLE
add the unconfined compression test compared with analytical solution

### PR DESCRIPTION
**Pull request by grubbymoon**
As titled.  The unconfined compression is divided into two steps. In early time, the drainage process is ongoing and the displacement behaviour is related to water pressure and solid properties. In late time, the drainage process is finished and the displacement of the porous media is only a result of solid properties. 

Todo:
- [ ] Update the docs.ogs.org with analytical solution and possibly a script.
- [x] Update ufz/ogs-data repository.

### CODE REVIEW

**Comment by endJunction** [comment]
Waiting for the docs.ogs.org to be updated with analytical solution and possibly a script.

**Comment by endJunction** [comment]
There are few things to finish before this can be reviewed, I'll set it to WorkInProgress until then.

File: Applications/CLI/Tests.cmake [hunk 1]
```diff
@@ -645,6 +645,31 @@ if(NOT OGS_USE_MPI)
...
+    AddTest(
+        NAME HydroMechanics_HML_square_1e2_unconfined_compression_long
+        PATH HydroMechanics/Linear/Unconfined_Compression_late
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS square_1e2.prj
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1e-3 RELTOL 1e-3
+        DIFF_DATA
+        UnconfinedCompressionAnalytical_1000s.vtu square_1e2_pcs_0_ts_100_t_1000.000000.vtu displacement_ana displacement
```
Comment (grubbymoon) [inline Applications/CLI/Tests.cmake]: no problem. i have changed the output name !

**Comment by grubbymoon** [comment]
test this please. Jenkins

**Comment by grubbymoon** [comment]
Jenkins, test this please.

File: Applications/CLI/Tests.cmake [hunk 2]
```diff
@@ -645,6 +645,31 @@ if(NOT OGS_USE_MPI)
...
+        WRAPPER time
+        TESTER vtkdiff
+        ABSTOL 1e-3 RELTOL 1e-3
+        DIFF_DATA
+        UnconfinedCompressionAnalytical_1s.vtu square_1e2_UC_early_pcs_0_ts_10_t_1.000000.vtu displacement_ana displacement
+    )
+    # HydroMechanics; Small deformation, linear poroelastic (unconfined compression long time)
+    AddTest(
+        NAME HydroMechanics_HML_square_1e2_unconfined_compression_long
+        PATH HydroMechanics/Linear/Unconfined_Compression_late
```
Comment (endJunction) [inline Applications/CLI/Tests.cmake]: What does the "early" and "late" mean? If it is some part of the experiment, please add info about what is "late" and what is early.

Comment (grubbymoon) [inline Applications/CLI/Tests.cmake]: I added the info in the first comment.

Comment (endJunction) [inline Applications/CLI/Tests.cmake]: It is good comment. Please add it to the source code.

**Comment by endJunction** [comment]
Waiting for the documentation page to be written.

**Pull request merged**